client/state/support/reducer.js

```diff
@@ -3,19 +3,42 @@
 /**
  * External dependencies
  */
+import debugFactory from 'debug';
 
+/**
+ * Internal dependencies
+ */
 import { combineReducers } from 'state/utils';
-import { SUPPORT_USER_ACTIVATE } from 'state/action-types';
+import { SUPPORT_SESSION_TRANSITION } from 'state/action-types';
+
+const debug = debugFactory( 'calypso:state:support:actions' );
+
+export const SESSION_NONE = 'none';
+export const SESSION_ACTIVE = 'active';
+export const SESSION_EXPIRED = 'expired';
 
-export function isSupportUser( state = false, { type } ) {
+export const SUPPORT_SESSION_STATE = {
+	NONE: 'none',
+	ACTIVE: 'active',
+	EXPIRED: 'expired',
+};
```

Comment:
did these come back accidentally during a rebase or in the additional work? I think they aren't needed, right?

Reply:
Seems like bad conflict resolution during rebase. Thanks for spotting this!